pstoreds/cyclic_batch.go

```diff
@@ -31,18 +31,18 @@ func newCyclicBatch(ds ds.Batching, threshold int) (ds.Batch, error) {
 
 func (cb *cyclicBatch) cycle() (err error) {
 	if cb.Batch == nil {
-		return errors.New("cyclic batch is closed")
+		return fmt.Errorf("cyclic batch is closed")
```

Comment:
`errors.New` is not wrong here, it should just be the standard library `errors` package.

Reply:
Ugh. I hate importing two error packages. But yeah, you're right.